source/material_model/rheology/visco_plastic.cc

```diff
@@ -208,8 +227,11 @@ namespace aspect
                   }
                   case composite:
                   {
-                    non_yielding_viscosity = (viscosity_diffusion * viscosity_dislocation)/
-                                             (viscosity_diffusion + viscosity_dislocation);
+                    if (use_minimum_creep_viscosity)
```

Comment:
I feel like it would be more consistent with the code structure to make this a new flow law separate from `composite`. Something like `composite_minimum_creep`.  That would save one additional input parameter and point out the difference between `composite` and `composite_minimum_creep`.